ledger/participant-local-store/src/test/lib/com/daml/platform/localstore/UserStoreTests.scala

```diff
@@ -500,6 +471,7 @@ trait UserStoreTests extends UserStoreSpecBase { self: AsyncFreeSpec =>
                   )
                 ),
               ),
+              identityProviderId = IdentityProviderId.Default,
```

Comment:
```suggestion
              identityProviderId = user.identityProviderId,
```
I'd make the handling of `identityProviderId` consistent throughout this unit test